app/src/main/java/com/fastlink/zekrapp/ui/favoriteScreen/FavoriteScreen.kt

```diff
@@ -1,4 +1,4 @@
-package com.fastlink.zekrapp.presentation
```

Comment:
UI is part of the presentation layer.

Reply:
We need to create a presentation layer? and move both the navigation and UI to it

Reply:
No you don't need it. But I thought you moved the ui out of the presentation layer. Keep it like that if you deleted the presentation layer.